### DISCUSSION TITLE
How to find out the reason why Log Agent installation failed.

### DISCUSSION THREAD

**Discussion by JoeShi** [aws-solutions/centralized-logging-with-opensearch#127] [Q&A]
I am trying to install the log agent from the web console. Some instances failed to install the log agent. How can I find out what are the reasons?

**Comment by James96315** [comment]
For Amazon Linux 2, please refer to this link to install:
https://aws-solutions.github.io/centralized-logging-with-opensearch/en/implementation-guide/resources/open-ssl/
